src/dotnet/APIView/APIViewWeb/Repositories/UnDeletableReviewException.cs

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+
+namespace APIViewWeb.Repositories
+{
+    internal class UnDeletableReviewException : Exception
```

Comment:
are there any tests we can add to validate this behavior?

Reply:
This will require a pull request review in test cosmos DB to test. Currently we don't have any tests around this area. I will try to add some test cases.

Reply:
I have added some tests for this.

Reply:
Thanks @chidozieononiwu